### PR TITLE
Adding files

### PR DESCRIPTION
**Pull request by jeffersonnnn**
This PR is the first step in the EVM docs upgrade. The metamask page is reqritten, the images are resized. select sections in the overview page are rewritten. Please review.

@kingpinXD

### CODE REVIEW

File: docs/guides/manual-setup.md [hunk 1]
```diff
@@ -1,36 +1,51 @@
 ---
 sidebar_position: 5
```
Comment (jianwel) [inline docs/guides/manual-setup.md:2]: Does this belong to another PR? better checkout main first when you start a new set of changes for a new PR

File: docs/dapp/01-overview.md [hunk 2]
```diff
@@ -14,21 +14,20 @@ Currently, developing on Findora EVM requires developers to connect to the Findo
...
 
 ### Testing and Automation
 
 Waffle is a library for compiling and testing smart contracts and Mars is a deployment manager. Waffle and Mars can be used together to write, compile, test, and deploy Ethereum smart contracts. 
 
 See the [Waffle & Mars](08-waffle-mars.md) guide for details.
 
-Hardhat is an Ethereum development environment that helps developers manage and automate repetitive tasks for smart contract and DApp development, and can be used in the truffle project. 
-
-See the [Hardhat](07-hardhat.md) guide for details.
```
Comment (kingpinXD) [inline docs/dapp/01-overview.md:31]: Not part of the changed file but might be a good idea to remove the word `Chainsafe Chainbridge` and replace with realto

File: docs/dapp/03-metamask.md [hunk 3]
```diff
@@ -1,13 +1,58 @@
...
+
+```jsx
+525
+```
+
+Beside the chain id, currency symbol is 
+
+```jsx
+FRA
+```
```
Comment (kingpinXD) [inline docs/dapp/03-metamask.md:42]: NIT : Lets put this whole section in a code block

File: docs/dapp/09-web3.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
+- ✅  &nbsp; eth_getBlockTransactionCountByHash
+- ✅  &nbsp; eth_getBlockTransactionCountByNumber
+- ✅  &nbsp; eth_getCode
+- ✅  &nbsp; eth_sendRawTransaction
+- ✅  &nbsp; eth_estimateGas
+- ✅  &nbsp; eth_getTransactionByHash
+- ✅  &nbsp; eth_getTransactionByBlockHashAndIndex
+- ✅  &nbsp; eth_getTransactionByBlockNumberAndIndex
+- ✅  &nbsp; eth_getTransactionReceipt
+- ✅  &nbsp; eth_getLogs
```
Comment (kingpinXD) [inline docs/dapp/09-web3.md:26]: Lets remove the Bullet points

File: docs/guides/manual-setup.md [hunk 5]
```diff
@@ -87,104 +124,169 @@ Key: {
...
+    -p 8545:8545 \
+    -p 26657:26657 \
+    -e EVM_CHAIN_ID=2152 \
+    --name findorad \
+    findoranetwork/findorad:latest node \
+    --ledger-dir /tmp/findora \
+    --tendermint-host 0.0.0.0 \
+    --tendermint-node-key-config-path="/root/.tendermint/config/priv_validator_key.json" \
+    --enable-query-service \
+    --enable-eth-api-service
```
Comment (kingpinXD) [inline docs/guides/manual-setup.md:200]: @jeffersonnnn  is this something you added ?

File: docs/dapp/03-metamask.md (hunk 3 above)
Comment (jianwel) [reply docs/dapp/03-metamask.md:42]: I guess it's OK to have separate fields which have separate "copy" buttons. However, it seems jsx is not the correct formatting syntax. Please make the color/style of each one consistent.

File: docs/guides/manual-setup.md (hunk 1 above)
Comment (jeffersonnnn) [reply docs/guides/manual-setup.md:2]: Yes it does. Added it in for the work that Jimmy is doing with the Manual setup guide for validators. I'll take it out and send it in a different PR.

File: docs/dapp/01-overview.md (hunk 2 above)
Comment (jeffersonnnn) [reply docs/dapp/01-overview.md:31]: New push to this branch will address this

File: docs/dapp/03-metamask.md (hunk 3 above)
Comment (jeffersonnnn) [reply docs/dapp/03-metamask.md:42]: I'll explore the options available to us for the formatting syntax in the next push

File: docs/dapp/09-web3.md (hunk 4 above)
Comment (jeffersonnnn) [reply docs/dapp/09-web3.md:26]: the empty boxes @kingpinXD

File: docs/guides/manual-setup.md (hunk 5 above)
Comment (jeffersonnnn) [reply docs/guides/manual-setup.md:200]: @kingpinXD this is work from Jimmy. As spoken earlier, i'll remove it

**Review by jianwel — APPROVED** [review]
